apps/dashboard/src/components/auth/customize-inbox-playground.tsx

```diff
@@ -168,23 +168,26 @@ function NotificationConfigSection() {
         <FormField
           control={control}
           name="subject"
-          render={({ field }) => (
-            <InputField size="fit">
-              <FormItem className="w-full">
-                <FormControl>
-                  <Editor
-                    singleLine
-                    indentWithTab={false}
-                    fontFamily="inherit"
-                    placeholder={capitalize(field.name)}
-                    id={field.name}
-                    extensions={extensions}
-                    value={field.value}
-                    onChange={field.onChange}
-                  />
-                </FormControl>
-              </FormItem>
-            </InputField>
+          render={({ field, fieldState }) => (
+            <FormItem className="w-full">
+              <FormControl>
+                <InputRoot hasError={!!fieldState.error}>
+                  <InputWrapper className="flex h-9 items-center justify-center px-1">
+                    <Editor
```

Comment:
I will refactor this in another PR to be FormEditorInput and avoid the duplication